autosklearn/pipeline/base.py

```diff
@@ -221,7 +221,7 @@ def set_hyperparameters(
             node_name, node = n_
 
             sub_configuration_space = node.get_hyperparameter_search_space(
-                feat_type=feat_type, dataset_properties=self.dataset_properties
+                feat_type=self.feat_type, dataset_properties=self.dataset_properties
```

Comment:
We should try to remove `feat_type` as an argument to this function as the `feat_type` should be already given in the constructor.